docs/faq/code-analysis/why-does-codacy-show-unexpected-coverage-changes.md

```diff
@@ -238,6 +238,9 @@ Consider an example pull request where Codacy shows the following metrics:
 
 However, since the proportion between the total number of covered and coverable lines across all files in the repository is now different, there can be a drop in the coverage variation for the pull request.
 
+!!! important
+    If you're using the [gate **Coverage variation is under**](../../repositories-configure/adjusting-quality-settings.md#gates), configure at least a **1% coverage drop margin** to ensure that developers aren't blocked while performing code refactors such as the one from this example.
```

Comment:
I'd say `-0.1%` should be enough to clear out corner cases. Whenever we saw this happening, the drop was of around 0.01%. `-1%` may be a lot for some people to consider it useful (yes, we use -1 ourselves, but that doesn't mean we should tell users to have "at least" that value for safety)

Reply:
I agree with this. Never saw the problem happening with a value different than -0.01

Reply:
It really depends on factors such as the size of the codebase, how many coverable lines there are and how many are covered, and the type of code refactors that are to be allowed. :+1: 

I'll change it to -0.10%.